index.html

```diff
@@ -35,12 +35,144 @@ <h3>I am a Software Developer</h3>
             <ul>
                 <li><a href="#"><i class="bi bi-github"></i></a></li>
                 <li><a href="#"><i class="bi bi-linkedin"></i></a></li>
-                <li><a href="#"><img src="NormalButton/Tertiary/Icons/Vector.svg" alt="peace button social media"></a>
+                <li><a href="#"><img src="vectorsImg/Tertiary/Icons/Vector.svg" alt="peace button social media"></a>
                 </li>
                 <li><a href="#"><i class="bi bi-twitter"></i></a></li>
-                <li><a href="#"><img src="NormalButton/Tertiary/Icons/Mvector.svg" alt="m button social media"></a></li>
+                <li><a href="#"><img src="vectorsImg/Tertiary/Icons/Mvector.svg" alt="m button social media"></a></li>
             </ul>
         </section>
+
+        <section class="work-section">
+            <h2>My Recent Works </h2>
+
```

Comment:
- [ ] Please make sure that you have the same line added here as in Figma design 

| Your design | Figma |
| ------------- | ------- |
| ![image](https://user-images.githubusercontent.com/74744787/164720827-49715d26-46a4-48b2-802b-24dff844609e.png) | ![image](https://user-images.githubusercontent.com/74744787/164720903-2c9bf672-4ba5-432c-8b77-7611015ed67d.png) |